README.md

```diff
@@ -6,52 +6,38 @@ This project provides a sandbox in which you can manually test your Launch exten
 
 For more information regarding Launch, please visit our [product website](http://www.adobe.com/enterprise/cloud-platform/launch.html).
 
-## Installing the Sandbox
-
-To use this project you will need to have [Node.js](https://nodejs.org/en/) installed on your computer. After you [download and install Node.js](https://nodejs.org/en/download/) you will also have access to the [npm](https://www.npmjs.com/) package manager for JavaScript. Your npm version will need to be at least 3.0.0. You can check the installed version by running the following command from a command line:
+## Running the Sandbox
 
+Before running the sandbox, you must first have [Node.js](https://nodejs.org/en/) installed on your computer. Your npm version (npm comes bundles with Node.js) will need to be at least 5.2.0. You can check the installed version by running the following command from a command line:
+                                                                                                      
 ```
 npm -v
 ```
 
-After you have installed Node.js on your machine, you will need to initialize your project. Create a folder for your project if you don't already have one. Inside the folder, run
+Once Node.js is installed, run the sandbox by executing the following command from the command line within your project's directory:
 
 ```
-npm init
+npx @adobe/reactor-sandbox
 ```
 
-You will need to provide the information requested on the screen. After this process is complete, you should have a file called `package.json` inside your folder.
-
-You will then need to install both @adobe/reactor-turbine and @adobe/reactor-sandbox and save them in your project's [`devDependencies`](https://docs.npmjs.com/files/package.json#devdependencies) by running
-```
-npm install @adobe/reactor-turbine @adobe/reactor-sandbox --save-dev
-```
-
-## Running the Sandbox
-
-To run the sandbox, run `node_modules/.bin/reactor-sandbox` from the command line within your project's directory. A server will be started at [http://localhost:3000](http://localhost:3000). If you navigate to that address in your browser, you will see the view sandbox which allows you to test your views. You'll need to have previously created a view in your extension for it to show up in the sandbox. You can switch between views you would like to test using the controls toward the top of the page.
+A server will be started at [http://localhost:3000](http://localhost:3000). If you navigate to that address in your browser, you will see the View Sandbox which allows you to test your views. You'll need to have previously created a view in your extension for it to show up in the sandbox. You can switch between views you would like to test using the controls toward the top of the page.
```

Comment:
Now we also have a server started on https://localhost:4000. Should we add this to the description?

Reply:
Yeah. Why aren't we able to do https://localhost:3000?

Reply:
Disregard. We talked about it in person.